docs/maui/views/AvatarView.md

```diff
@@ -0,0 +1,360 @@
+
+`AvatarView` defines the following properties:
+
+- `BackgroundColor`, of type `Color`, describes the background color of the AvatarView. The default value of this property is White.
+- `BorderColor`, of type `Color`, describes the border color of the AvatarView.
+- `BorderWidth`, of type `double`, defines the width of the AvatarView's border.  The default value for this property is 1.0.
+- `CharacterSpacing`, of type `double`, defines the spacing between characters of the AvatarViews's text.
+- `CornerRadius`, of type `CornerRadius`, which defines the corner radius of the `AvatarView`. This property can be set to a single `double` uniform corner radius value, or a `CornerRadius` structure defined by four `double` values that are applied to the top left, top right, bottom left, and bottom right of the `AvatarView`.  The default value of this property is a device-independent unit of 24.
+- `FontAttributes`, of type `FontAttributes`, determines text style.
+- `FontAutoScalingEnabled`, of type `bool`, defines whether the AvatarView text will reflect scaling preferences set in the operating system. The default value of this property is `true`.
+- `FontFamily`, of type `string`, defines the font family.
+- `FontSize`, of type `double`, defines the font size.  The default value for this property is a device-independent unit of 14.
+- `ImageSource`, of type `ImageSource`, specifies an image to display as the content of the AvatarView.
+- `Padding`, of type `Thickness`, represents the distance between the avatar border and its text or image element.
+- `Text`, of type `string`, defines the text displayed as the content of the AvatarView.  This default value for this property is '?'.
+- `TextColor`, of type `Color`, describes the color of the AvatarView's text.
+- `TextTransform`, of type `TextTransform`, defines the casing of the AvatarView's text.
+
+These properties are backed by `BindableProperty` objects, which means that they can be targets of data bindings and styled.
+
+For information about specifying fonts on an `AvatarView`, see [Fonts](/dotnet/maui/user-interface/fonts.md).
+
+For information about specifying shadows on an `AvatarView`, see [Shadows](/dotnet/maui/user-interface/shadow.md)
+
+
+> [!IMPORTANT]
+> `AvatarView` will use the default `WidthRequest` and `HeightRequest` of 48 unless the size of the `AvatarView` is constrained by its layout, or the `HeightRequest` or `WidthRequest` property of the `AvatarView` is specified. The `WidthRequest` and `HeightRequest` properties are measured in device-independent units.
+
+## Create an AvatarView
+
+The following example shows how to create an `AvatarView`:
+
+```xaml
+<views:AvatarView
+	BackgroundColor="Blue"
+	BorderColor="Red"
+	BorderWidth="1"
+	CornerRadius="24, 0, 0, 24"
+	FontAttributes="Bold"
+	FontSize="Large"
+	HeightRequest="48"
+	Padding="0"
+	Text="ZS"
+	TextColor="Yellow"
+	WidthRequest="64" />
```

Comment:
Given some of this is expanded on in later sections could/should we go with a minimal set of properties here?

Reply:
@bijington As later sections are targeted towards giving examples specific to a property, I thought it would be a promising idea to give an expanded example.  We could however do this later in the document (under an "Example" heading?) and have this current one as a minimal set of properties.  More than happy to go with your experience and guidance here.

Reply:
I am learning as I go too here so this conversation is good. @brminnick and @jfversluis what do you think do having a cutdown example first and then moving out to showing an advanced scenario at the end?

Reply:
As I am making changes to the document, your initial reaction (@bijington) I feel is right, and it should be a minimal set of properties.  We provide extensive examples in the code repository and provide a link to this later in the document.  I am, as always, open to discussions and direction from @bijington , @jfversluis and others.

Reply:
Yeah I like what you have done. Thanks